### ISSUE TITLE
Target bundle of current build by babel

### ISSUE THREAD

**Issue by mariusrak** [meteor/meteor-feature-requests#326]
I'm creating plugin for babel which should compile files differently based on whether resulting package is meant for server or client. Therefore Id need a way to know what is a target for the package.

Suggested solution:
babel object, which is parameter for function returning babel plugin could contain some field with target. E.g. `babel.target = 'client'` or if this is not possible then maybe `process.args` or `Meteor.packageTarget`

I didn't find any possibility for this yet.

**Comment by mariusrak** [comment]
After talk with @loganfsmyth proposed solution is that the flag would be accessible via Babel 7 `caller` feature, his proposed solution would be 

```
function isMeteorServer(caller) {
 return caller && caller.name === "meteor" && caller.target === "server";
}

module.exports = function(api) {
 const isServer = api.caller(isMeteorServer);
 // the rest of your plugin
};
```

So now it's up to meteor to provide he information for babel.

**Comment by benjamn** [comment]
I believe this has been implemented and shipped in Meteor 1.8!
